jormungandr/src/rest/v0/logic.rs

```diff
@@ -228,11 +233,11 @@ async fn create_stats(context: &Context) -> Result<Option<NodeStats>, Error> {
         last_block_time: SystemTime::from(tip.time()).into(),
         last_block_tx: block_tx_count,
         last_received_block_time: stats.slot_start_time().map(SystemTime::from),
-        peer_available_cnt: nodes_count.available_count,
+        peer_available_cnt,
         peer_connected_cnt: stats.peer_connected_cnt(),
-        peer_quarantined_cnt: nodes_count.quarantined_count,
-        peer_total_cnt: nodes_count.all_count,
-        peer_unreachable_cnt: nodes_count.not_reachable_count,
+        peer_quarantined_cnt,
+        peer_total_cnt,
+        peer_unreachable_cnt: 0, // FIXME
```

Comment:
Why is that?

Reply:
Those definitions were previously used by poldercast, I don't know if we want to keep them all or not and I did not want to break the rest api. For example, in this case, what would `peer_unreachable` mean? If it means nodes we could not connect to it seems pretty similar to the quarantined ones atm.